bc_obps/bc_obps/api.py

```diff
@@ -1,12 +1,20 @@
 from ninja import NinjaAPI, Swagger
 from common.api import router as common_router
 from registration.api import router as registration_router
+from ninja.errors import ValidationError
 
 # Docs: https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/
 # Filtering is case sensitive matching the filter expression anywhere inside the tag.
 api = NinjaAPI(
     title="BCIERS API", docs=Swagger(settings={"filter": True, "operationsSorter": "method", "tagsSorter": "alpha"})
 )
 
+
```

Comment:
This was taken out at some point--can we put it back in and leave it until we're done form development? It helps debug ninja schema errors

Reply:
Is this still helpful? I thought our `handle_exception` decorator did the same. If not sorry for the hassle.

Reply:
It's helpful when getting lots of 422 (like when making forms!) because you can see which field is the problem. Handle_exception doesn't do this

Reply:
We can remove this once we merge this ticket: https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=&pane=issue&itemId=69067820